From 4f49f71a8b2c0852d3dfcd990d0b8bb61b1974c8 Mon Sep 17 00:00:00 2001
From: Florian Weimer <fweimer@redhat.com>
Date: Thu, 15 Dec 2022 20:09:35 +0100
Subject: [PATCH] fcitx-gclient: Export fcitx_client_enable_ic,
 fcitx_client_close_ic (#521)

Git-commit: 4f49f71a8b2c0852d3dfcd990d0b8bb61b1974c8

These functions are used from fcitx-fbterm.  Adding function
prototypes to the installed header file seems to be the cleanest way
to ensure that fcitx-fbterm does not rely on implicit function
declarations (which are likely to result in errors with future
compilers by default).
---
 src/lib/fcitx-gclient/fcitxclient.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/lib/fcitx-gclient/fcitxclient.h b/src/lib/fcitx-gclient/fcitxclient.h
index 52f0c958..8942111c 100644
--- a/src/lib/fcitx-gclient/fcitxclient.h
+++ b/src/lib/fcitx-gclient/fcitxclient.h
@@ -88,6 +88,9 @@ void fcitx_client_set_capacity(FcitxClient *self, guint flags);
 
 void fcitx_client_reset(FcitxClient *self);
 
+void fcitx_client_enable_ic(FcitxClient *self);
+void fcitx_client_close_ic(FcitxClient *self);
+
 G_END_DECLS
 
 #endif // CLIENT_IM_H
-- 
2.45.1

